content/course-section/video-lectures.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Video Lectures"
+date: 2019-11-07T11:18:00-05:00
+draft: false
+content: "<p>These video lectures of Professor Gilbert Strang teaching 18.06 were recorded live in the Fall of 1999. Support for the video production was provided by the Lord Foundation of Massachusetts under a grant to the MIT Center for Advanced Educational Services.</p>"
+menu: 
+    main:
+        name: "Video Lectures"
+---
```

Comment:
where are the videos?

Reply:
I figured we would want to handle video galleries in a way that wasn't just HTML smashed into the page content, so I left these out for now.